tests/integration/scripts/load_ebpf.c

```diff
@@ -0,0 +1,82 @@
+#include <linux/bpf.h>
```

Comment:
Have you tried using `bpftool` or `bpftrace` to trigger BPF events? It would simplify maintenance by reducing the amount of test code needed.

Reply:
Its possible, but in order to avoid downloading/compiling both `bpftool` and `bpftrace` I've combined both trigger in the C program. Do you think its better avoiding coding when possible? I'm OK with both solutions.

Trigger `security_bpf_prog` where name `sys_enter_execv`
> sudo bpftrace -e 'tracepoint:syscalls:sys_enter_execve { printf("Process: %s\n", comm); }'

Trigger `security_bpf_map` where name `ebpf_prog_test`
> sudo bpftool map create /sys/fs/bpf/ebpf_map_test type hash key 4 value 8 entries 100 name ebpf_map_test